<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1404</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 170, 127);</string>
  </property>
  <widget class="QVTKWidget" name="meshwidget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1021</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-width: 1px;
border-color: black;
border: 1px solid blue;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="device_Button">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>30</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>连接设备</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detection_Button">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>205</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>检测煤堆</string>
   </property>
  </widget>
  <widget class="QPushButton" name="compute_Button">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>374</y>
     <width>221</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>计算体积</string>
   </property>
  </widget>
  <widget class="QLabel" name="volume_label">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>540</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>体积：</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_volume_label">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>540</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
background-color: rgb(255, 255, 255);
border-width: 1px;
border-color: blue;
border: 1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="mass_label">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>620</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>质量：</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_mass_label">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>620</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
background-color: rgb(255, 255, 255);
border-width: 1px;
border-color: blue;
border: 1px solid white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>540</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>m³</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1330</x>
     <y>620</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>吨</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>500</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 170, 255);</string>
   </property>
   <property name="text">
    <string>当前煤堆密度为1.3吨/m³</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
